src/App.tsx

```diff
@@ -1,18 +1,32 @@
 import React from 'react';
 import { ThemeProvider } from 'styled-components';
-import { AppHeader, AppLogo, AppWrapper } from './components';
+import { AppHeader, AppHeaderLeft, AppHeaderCenter, AppHeaderRight, AppHeaderLogoLinkWrapper, AppLogo, AppHeaderNews, AppHeaderProof, AppWrapper } from './components';
 import { defaultTheme } from './theme/defaultTheme';
 
 import logo from './images/gardener-logo_horizontal.svg';
 import RequestList from './requests/RequestList';
 import ServerStatus from './status/ServerStatus';
+import CallForm from './customerCalls/CallForm';
+
+const url = 'https://gardeneroracle.io/';
```

Comment:
better to have such urls in some kind of config in different file. For sure in project structure should be something like config.ts also such url can be places in .evn file

Reply:
created config.ts for urls